CHANGELOG.md

```diff
@@ -7,7 +7,7 @@
 * Protect CKeyHolderStorage via mutex
 * Fix Boost 1.66 Compatibility
 * Net Overhaul and BTC Inlining
-* Bump Versions/Protocol
+* Bump Versions/Protocol (Updated Dynodes must get an fresh "Start" Signal with the new Binaries)
```

Comment:
"Updated Dynodes must get a fresh "Start" Signal from version 2.3"

Reply:
I made it "new Binaries" because its something that must be done at every Proto change, thats why i made the comment generally and not version specified

Reply:
can you change the "an" to an "a" and then i will merge ;)